esp/esdllib/README.md

```diff
@@ -621,13 +627,20 @@ This reference section covers details about the ESDL Scripting conventions, scri
 
 ## Common Attributes
 
-These attributes shared by many ESDL Script operations are explained once here rather than under each operation's section.
+Some ESDL Script language XML elements share attributes that have the same meaning or usage. This section explains attributes common to many of the operations and the entry point elements.
+
+### Operations ###
 
 | Name      | Usage Notes |
 |-----------|-------------|
 | @name     | Interpreted as a scalar value so quoting is not required. XPath expressions are not supported. String identifier used in trace output. Use is optional, but strongly encouraged. |
-| @target   | XPath expression that evaluates to the target of an operation or destination of an entry point's output document node. As an _entry point_ attribute it has these additional characteristics: <br><br> 1. Any [Implicit Variables](#implicit-variables) can be used. <br> 2. The XPath must exist at runtime or an exception will be thrown. [A future release](https://track.hpccsystems.com/browse/HPCC-25844) will ensure the path exists without throwing an exception. |
 | @value    | Interpreted as a scalar value so quoting is not required. XPath expressions are not supported. |
```

Comment:
I'm not sure @value is common enough to be included here. If it is, XPath expressions may be required (see `set-value` which treats `@value` as a synonym of `@select`.

Reply:
I agree.